Extensions/super_markdown_editor.js

```diff
@@ -0,0 +1,262 @@
+//* TITLE Super Markdown Editor **//
+//* VERSION 1.0 REV A **//
+//* DESCRIPTION Add live previews and other functionality to the Markdown post editor **//
+//* DEVELOPER new-xkit **//
+//* FRAME false **//
+//* BETA false **//
```

Comment:
I enjoy being able to point out to users that this is a beta release if they encounter bugs. This is sufficiently complicated enough that introducing it to users as a potentially error-prone beta makes sense.